include/revng/Model/Binary.h

```diff
@@ -124,6 +124,13 @@ class model::Binary : public model::generated::Binary {
   model::TypePath
   getPrimitiveType(PrimitiveTypeKind::Values V, uint8_t ByteSize) const;
 
+  model::QualifiedType getPointerTo(const model::QualifiedType &Type) const {
+    QualifiedType Result = Type;
+    Result.Qualifiers().insert(Result.Qualifiers().begin(),
+                               model::Qualifier::createPointer(Architecture()));
+    return Result;
+  }
```

Comment:
In the layout-generating code, there's no access to the binary (it can't introduce new types anyway, there's no need to let it have it), as such I needed a version of `getPointerTo` that doesn't require one.

Writing
```cpp
model::QualifiedType MyPointer MyQualifiedType.getPointerTo(MyArchitecture);
```
definitely bits inserting qualifiers manually.

Not sure how this stacks with the the rework though. If you have incompatibility concerns, I can settle for a temporary manual qualifier insertion + a todo.

Reply:
Whoops, sorry, I just realized what was going on 😅. This commit went in with #241 and here is just a remainder of a botched rebase. My bad.

(Don't mind the previous message in in this thread, I was misunderstanding the situation)

Reply:
It's gone.